### ISSUE TITLE
NTR: Abdominal imaging: fecolith (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9027]
New term request Abdominal imaging: fecolith (nadea3_190227163859): Appendicolith; fecaloma; calcification in intestine; abdomen; fecalith; faecalith; coprolith; stercolith  UMLS: 0333033

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000142
- label: Fecolith
- parents: [Abnormal large intestine morphology (HP:0002250)](https://hpo.jax.org/app/browse/term/HP:0002250)
- pmid: PMID:8906682|PMID:37706403
- def: A mass of accumulated hardened fecal matter.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9027
- synonyms: Stercolith(NONE)|Coprolith(NONE)|Fecaloma(NONE)
